Gemfile

```diff
@@ -3,6 +3,8 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'mysql2'
+gem 'pg'
+gem 'sqlite3'
```

Comment:
Can anyone explain to me why this is in the `Gemfile` and not in the `gemspec`? I'm not sure I understand which is for what